app/src/main/AndroidManifest.xml

```diff
@@ -21,14 +21,14 @@
   <uses-permission android:name="android.permission.GET_TASKS"/>
 
   <permission
-    android:name="org.gdg.frisbee.android.permission.MAPS_RECEIVE"
+    android:name="${applicationId}.permission.MAPS_RECEIVE"
```

Comment:
Thank you but Travis is failing because it cannot find the commit. I have to fix that.